mod_reforged/hooks/skills/skill.nut

```diff
@@ -5,4 +5,57 @@
 	{
 		return this.isAttack() && !this.isRanged() && this.getBaseValue("ActionPointCost") <= 4 && this.getBaseValue("MaxRange") == 1;
 	}
+
+	local getHitFactors = o.getHitFactors;
+	o.getHitFactors = function( _targetTile )
+	{
+		local ret = getHitFactors(_targetTile);
+		for (local index = (ret.len() - 1); index >= 0; index--)	// We traverse it in reverse because we want to remove entries during the loop
+		{
+			switch (ret[index].text)
+			{
+				// For these entries we switch out the Icon for a better one
+				case "Immune to stun":
+				case "Immune to being rooted":
+				case "Immune to being disarmed":
+				case "Immune to being knocked back or hooked":
+				ret[index].icon = "ui/tooltips/warning.png";	// These cases should be a warnings and not hide in-between hit-chance lines
+					break;
+
+				// We remove these entries from ever displaying because they are mostly redundant and take away space
+				case "Fast Adaption":		// This should be handled by the perk itself if it really feels like this information is important
```

Comment:
Anything that affects directly affects hit-chance should show up in this tooltip. Therefore, Fast Adaptation and Nighttime specifically are important in my opinion. Nighttime especially as it is something that is negated by a certain items in the game (and maybe more from mods) - showing when it is applying and when not is relevant information.

Reply:
Nighttime is already listed under effects and it is directly affecting your attributes in real time which you can also see in the tooltip. That is my cut-off point for this PR, that's where I chose to draw the line whether to remove a vanilla entry.

Fast Adaption should be shown, but by the perk itself and not by the `skill.nut` class in a hardcoded way.